src/main/java/com/google/devtools/build/lib/analysis/constraints/RuleContextConstraintSemantics.java

```diff
@@ -882,159 +882,4 @@ public static IncompatibleCheckResult checkForIncompatibility(ConfiguredTarget t
     return IncompatibleCheckResult.create(
         target.get(IncompatiblePlatformProvider.PROVIDER) != null, target);
   }
-
```

Comment:
Sanity check: all this removed logic remains present in the other files?

Reply:
Yes. Now in a file called `RuleConstraintSemantics.java`. I apologize for the name. I will rename it.

The reason it's removed here is that we no longer have access to a `RuleContext`.